swarm/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.4
+WORKDIR /go/src/github.com/docker
+RUN git clone https://github.com/smothiki/swarm
```

Comment:
We cloned `swarm` to `deis/swarm` so we could use that codebase instead. There shouldn't be any `smothiki`s here.